src/components/FutureTripCard.js

```diff
@@ -1,22 +1,34 @@
 import styled from 'styled-components';
 import { FiTrash } from 'react-icons/fi';
+import Modal from './Modal';
+import { useState } from 'react';
 
-export default function FutureTripCard({ trip, onDeleteCard }) {
+export default function FutureTripCard({ trip, onDelete, id }) {
   var dayjs = require('dayjs');
```

Comment:
😱 a `var`?! 🤯 

😆 

And `require`? 

I would recommend adding this as a normal import:
```js
import dayjs from 'dayjs'
```
Should work the same :)